mock_service.go

```diff
@@ -31,33 +32,58 @@ type MockEndpoint struct {
 	// RequestBody     string            `json:"requestBody"`
 }
 
-func New(createMockEndpoint string) *MockService {
+type MockEndpointSeries []MockEndpoint
+
+func New(mockCreatorEndpoint string) *MockService {
 	return &MockService{
-		createMockEndpoint: createMockEndpoint,
-		mockEndpoints:      map[string]map[string]MockEndpoint{},
+		mockCreatorEndpoint:  mockCreatorEndpoint,
+		mockedEndpoints:      map[string]map[string]MockEndpoint{},
 	}
 }
 
 func (m *MockService) ServeHTTP(w http.ResponseWriter, req *http.Request) {
-	if req.Method == http.MethodPost && req.URL.Path == m.createMockEndpoint {
+	if req.Method == http.MethodPost && req.URL.Path == m.mockCreatorEndpoint {
 		m.serveRegistrationHTTP(w, req)
 		return
 	}
 
 	m.serveMockHTTP(w, req)
 }
 
+func (m *MockService) PreloadEndpointsFromConf(confFilePath string) (error, bool){
+    if  len(m.mockCreatorEndpoint) < 1 {
```

Comment:
Is the idea to block the possibility of the `mockEndpoint` being an empty string? If it is, it might be better to protect this in the `New` method instead. 

Its also a little clearer if we test for an empty string eg. `strings.Trim(m.mockRegistrationEndpoint) == ""` or something similar.